mapper/thin_edge_json/src/measurement.rs

```diff
@@ -151,7 +151,7 @@ pub trait FlatMeasurementVisitor {
 /// ```
 pub trait GroupedMeasurementVisitor {
     /// Error type specific to this way of collecting measurements
-    type Error;
+    type Error: std::error::Error + std::fmt::Debug;
```

Comment:
does this mean Error could be either sted::error/ std::fmt:Debug?.

Reply:
This type specification should also be added to the `FlatMeasurementVisitor` associated `Error` type.